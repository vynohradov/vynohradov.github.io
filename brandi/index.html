<!DOCTYPE html>
<html lang="en">
<head>
    <meta name="viewport" content="width=device­width; initial­scale=1.0">

    <link rel="stylesheet" href="css/style.css"/>
    <meta charset="UTF-8">
    <title></title>

</head>
<body>
<div id="webpage">
    <header id="header_fixed">
        <div class="header_wrapper">
            <div id="logo">
                <span>brandi</span>
                <span>|</span>
                <span>I'am your tag line</span>
            </div>
            <nav class="hmenu">
                <ul>
                    <li><a href="#">home</a></li>
                    <li><a href="#">features</a></li>
                    <li><a href="#">works</a></li>
                    <li><a href="#">team</a></li>
                    <li><a href="#">contact</a></li>
                </ul>
            </nav>
        </div>

    </header>
    <section id="header_main">
        <div class="vdots">
            <div class="circle active_circle"></div>
            <div class="circle"></div>
            <div class="circle"></div>
        </div>
        <div id="title">
            <h1>meet <span>brandi!</span></h1>
            <img src="img/header_1/hr.png" alt=""/>
            <p>We are a team of professionals</p>
            <img src="img/header_1/hr.png" alt=""/>
        </div>
        <div id="snetworks">
            <div class="icons twitter"><a href="#"></a></div>
            <div class="icons facebook"><a href="#"></a></div>
            <div class="icons google_plus"><a href="#"></a></div>
            <div class="icons dribble"><a href="#"></a></div>
        </div>
    </section>
    <section id="section_features_1">
        <h2>features</h2>
        <img class="divider" src="img/hr.png" alt=""/>
        <br>
        <div class="container">
            <div class="category ">
                <div class="icon pencil"></div>
                <div class="description">
                    <h4>branding</h4>
                    <p>Lorem  fsipsum dolor sit amet,
                        consectetur adipisicing elit.
                        Quaerat quod quos soluta voluptate,
                        voluptatem.</p>
                </div>
            </div>
            <div class="category ">
                <div class="icon pen"></div>
                <div class="description">
                    <h4>development</h4>
                    <p>Lorem  fsipsum dolor sit amet,
                        consectetur adipisicing elit.
                        Quaerat quod quos soluta voluptate,
                        voluptatem.</p>
                </div>
            </div>
            <div class="category ">
                <div class="icon megafon"></div>
                <div class="description">
                    <h4>consulting</h4>
                    <p>Lorem  fsipsum dolor sit amet,
                        consectetur adipisicing elit.
                        Quaerat quod quos soluta voluptate,
                        voluptatem.</p>
                </div>
            </div>

        </div>
        <div class="hdots">
            <div class="circle_inv active_circle_inv"></div>
            <div class="circle_inv"></div>
            <div class="circle_inv"></div>
        </div>
    </section>
    <section id="section_works_2">
        <h2>works</h2>
        <img class="divider" src="img/hr.png" alt=""/>
        <br>
        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, <br>
            nam nemo nesciunt nostrum odit omnis sed sunt, voluptas.</p>
        <div class="filter">
            <ul>
                <li class="picked_filter"><a href="#">all</a></li>
                <li><a href="#">branding</a></li>
                <li><a href="#">web</a></li>
                <li><a href="#">logo design</a></li>
                <li><a href="#">photography</a></li>
            </ul>
        </div>
        <div class="images">
            <div class="image photo_1"></div>
            <div class="image photo_2"></div>
            <div class="image photo_3"></div>
            <div class="image photo_4"></div>
            <div class="image photo_5"></div>
            <div class="image photo_6"></div>
            <div class="image photo_7"></div>
            <div class="image photo_8"></div>
        </div>
    </section>
    <section id="section_team_3">
        <h2>meet our team</h2>
        <img class="divider" src="img/hr.png" alt=""/>
        <br>
        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit, <br>
            nam nemo nesciunt nostrum odit omnis sed sunt, voluptas.</p>
            <div class="stuff">
                <div class="employee">
                    <div class="photo john">
                        <div class="wrap">
                            <h4>Lorem ipsum dolor sit amet, consectetur</h4>
                            <p class="description">
                                Lorem ipsum dolor sit amet,
                                consectetur adipisicing elit.
                                Expedita iure non odio reiciendis.
                                Ab alias aspernatur.
                            </p>
                            <div class="sn">
                                <div class="sn_icon sn_twitter"><a href="#"></a></div>
                                <div class="sn_icon sn_facebook"><a href="#"></a></div>
                                <div class="sn_icon sn_google_plus"><a href="#"></a></div>
                            </div>
                        </div>
                    </div>
                    <h4 class="name">john filmr doe</h4>
                    <h5 class="speciality">managing director</h5>
                </div>
                <div class="employee">
                    <div class="photo chrystine">
                        <div class="wrap">
                            <h4>Lorem ipsum dolor sit amet, consectetur</h4>
                            <p class="description">
                                Lorem ipsum dolor sit amet,
                                consectetur adipisicing elit.
                                Expedita iure non odio reiciendis.
                                Ab alias aspernatur.
                            </p>
                            <div class="sn">
                                <div class="sn_icon sn_twitter"><a href="#"></a></div>
                                <div class="sn_icon sn_facebook"><a href="#"></a></div>
                                <div class="sn_icon sn_google_plus"><a href="#"></a></div>
                            </div>
                        </div>
                    </div>
                    <h4 class="name">chrystine hineu</h4>
                    <h5 class="speciality">lead designer</h5>
                </div>
                <div class="employee">
                    <div class="photo martin">
                        <div class="wrap">
                            <h4>Lorem ipsum dolor sit amet, consectetur</h4>
                            <p class="description">
                                Lorem ipsum dolor sit amet,
                                consectetur adipisicing elit.
                                Expedita iure non odio reiciendis.
                                Ab alias aspernatur.
                            </p>
                            <div class="sn">
                                <div class="sn_icon sn_twitter"><a href="#"></a></div>
                                <div class="sn_icon sn_facebook"><a href="#"></a></div>
                                <div class="sn_icon sn_google_plus"><a href="#"></a></div>
                            </div>
                        </div>
                    </div>
                    <h4 class="name">martin matrone</h4>
                    <h5 class="speciality">lead developer</h5>
                </div>
                <div class="employee">
                    <div class="photo steve">
                        <div class="wrap">
                            <h4>Lorem ipsum dolor sit amet, consectetur</h4>
                            <p class="description">
                                Lorem ipsum dolor sit amet,
                                consectetur adipisicing elit.
                                Expedita iure non odio reiciendis.
                                Ab alias aspernatur.
                            </p>
                            <div class="sn">
                                <div class="sn_icon sn_twitter"><a href="#"></a></div>
                                <div class="sn_icon sn_facebook"><a href="#"></a></div>
                                <div class="sn_icon sn_google_plus"><a href="#"></a></div>
                            </div>
                        </div>
                    </div>
                    <h4 class="name">steve flaulkin</h4>
                    <h5 class="speciality">sr.UI designer</h5>
                </div>
            </div>
        <div class="hdots clearfix">
            <div class="circle_inv active_circle_inv"></div>
            <div class="circle_inv"></div>
            <div class="circle_inv"></div>
        </div>
    </section>
    <section id="section_facts_4">
        <h2>some fun facts</h2>
        <img class="divider" src="img/hr_inv.png" alt=""/>
        <br>

        <div class="widgets">
            <div class="widget">
                <div class="image clock"></div>
                <p class="counter">3200</p>

                <p class="description">hours of work</p>
            </div>
            <div class="widget">
                <div class="image people"></div>
                <p class="counter">120</p>

                <p class="description">satisfied clients</p>
            </div>
            <div class="widget">
                <div class="image rocket"></div>
                <p class="counter">360</p>

                <p class="description">project delivered</p>
            </div>
            <div class="widget">
                <div class="image cup"></div>
                <p class="counter">42</p>

                <p class="description">awards won</p>
            </div>
        </div>
    </section>
    <section id="section_discuss_5">
        <h2>let's disscus</h2>
        <img class="divider" src="img/hr.png" alt=""/>
        <br>
        <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>

        <div class="container">
            <div class="address">
                <h3>Cras at ultrices erat,<br> sed vulputate!</h3>
                <p>
                    2345 Setwant natrer, 1234,<br>
                    Washington. United States.<br>
                    (401) 1234 567<br>
                    hello@brandi.com<br>
                    www.brandi.com
                </p>
            </div>
            <form action="#">
                <h3>Sey Hello!</h3>
                <input class="name" placeholder="name" type="text"/>
                <input placeholder="e-mail" type="email"/><br>
                <input class="msg" placeholder="message" type="text"/><br>
                <button type="submit">Send message</button>
            </form>
            <div class="sn_link">
                <div class="sn_icons behance"><a href="#"></a></div>
                <div class="sn_icons twitter"><a href="#"></a></div>
                <div class="sn_icons dribble"><a href="#"></a></div>
                <div class="sn_icons forrst"><a href="#"></a></div>
                <div class="sn_icons google_plus"><a href="#"></a></div>
            </div>
        </div>
    </section>
    <section id="section_map_6"></section>
    <footer>
        <div class="container">
            <div class="cell cat_logo">
                <span>brandi</span><br/>

                <p>
                    Eusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
                    consequat</p>
            </div>
            <div class="cell subscribe">
                <h5>subscribe</h5>

                <form action="#">
                    <input type="email"/>
                    <button></button>
                </form>

                <p>eusmod tempor incididunt ut labore et dolore magna aliqua. </p>
            </div>
            <div class="cell explore">
                <h5>explore</h5>

                <div class="col">
                    <a href="#">envato</a>
                    <a href="#">themecurve</a>
                    <a href="#">kreativeshowcase</a>
                    <a href="#">freebiecurve</a>
                </div>
                <div class="col">
                    <a href="#">themeforest</a>
                    <a href="#">microsoft</a>
                    <a href="#">google</a>
                    <a href="#">yahoo</a>
                </div>
            </div>
            <div class="cell rights">
                <p>Created with <img src="img/footer/heart.png" alt=""/> by
                    <span>themecurve</span>,<br>
                exclusive for kreativeshowcase
                </p>
                <p>© 2014 Brandi. All Rights Reserved.</p>
            </div>
        </div>
    </footer>

</div>
</body>
</html>