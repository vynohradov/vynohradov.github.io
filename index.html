<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Document</title>
</head>

<body>

    <a href="./brandi/index.html">Brandi</a>
    <a href="./phototime/index.html">Phototime</a>
    <a href="./golden//index.html">Golden</a>


</body>

</html>