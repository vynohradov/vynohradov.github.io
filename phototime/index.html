<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<link rel="stylesheet" href="css/style.css">
	<title>Phototime</title>
</head>
<body>
	<div class="wrapper">
		<header class="first_section">
				<nav class="navigation">
					<ul class="left_menu">
						<li><a href="#">HOME</a></li>
						<li><a href="#">PHOTOAPP</a></li>
					</ul>
						<a href="#"><img class="logo" src="img/icons/logo.png" alt=""></a>
					<ul class="right_menu">
						<li><a href="#">DESIGN</a></li>
						<li><a href="#">DOWNLOAD</a></li>
					</ul>
				</nav>
			<div class="container">
				<p>YOUR LIFE , A PHOTO</p>
				<p>print your life in a simple photo</p>
				<button class="button">GET STARTED</button>
				<div class="favicons">
					<a href="#"><img src="img/icons/favico_twitter.png" alt=""></a>
					<a href=""><img src="img/icons/favico_flickr.png" alt=""></a>
					<a href=""><img src="img/icons/favico_dribble.png" alt=""></a>
				</div>				
			</div>
		</header>
		<section class="second_section">
			<div class="s2_text">
				<h3>RAPPRESENT YOUR LIFE WITH A	SIMPLE PHOTO</h3>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.<br>Nobis totam praesentium labore, alias suscipit asperiores. <br>Alias, doloribus veniam voluptatibus expedita, <br>quasi veritatis. Illum commodi tempore<br> corporis impedit, nobis nulla.</p>
			</div>
			<button class="btn_pos">GET STARTED</button>
		</section>
		<section class="third_section">
			<div class="s3_text">
				<h3>DESIGN</h3>
				<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.<br>Nobis totam praesentium labore, alias suscipit asperiores. <br>Alias, doloribus veniam voluptatibus expedita, <br>quasi veritatis. Illum commodi tempore<br> corporis impedit, nobis nulla.</p>
			</div>
			<button class="s3_button">Learn more</button>
			<p class="counter">01/05</p>
			<div class="angle">
				<img src="img/angle_left.png" alt="">
				<img src="img/angle_right.png" alt="">	
			</div>			
		</section>
		<section class="fourth_section">
			<div class="arrows">
				<img src="img/angle_left.png" alt="">
				<img src="img/angle_right.png" alt="">
			</div>
			<div class="sub_container">
				<div class="pseudo_pict">
					<img src="img/icons/logo_green.png" alt="">
					<h3>OUR COMMUNITY</h3>
					<p>Lorem ipsum dolor <br>sit amet, consectetur <br>adipisicing elit!</p>
				</div>
				<img class="gallery" src="img/pic_cars.jpg" alt="">
				<img class="gallery" src="img/pic_tree.jpg" alt="">			
			</div>

		</section>
		<section class="fiveth_section">
			<h2>SUBSCRIBE TO US COMMUNITY</h2>
			<form action="#">
				<input type="email" placeholder="YOUR E-MAIL">
				<button></button>
			</form>
		</section>

		<section class="sixth_section">
			<h2>DOWNLOAD IT</h2>
			<p>
				Lorem ipsum dolor sit amet, consectetur adipisicing elit.
				Ad aliquam aut blanditiis, consectetur <br>deleniti dolorem
				doloremque dolorum ea eaque eius enim expedita.consectetur deleniti dolorem<br> doloremque dolorum ea eaque eius enim expedita.
			</p>
			<form action="#">
				<button id="appstore">APPLE STORE <div></div></button>
				<button id="playstore">PLAY STORE <div></div></button>
			</form>
		</section>
		<footer>
			<div class="footer_info">
				<h3>
					<a href="#">CREDITS</a> |
					<a href="#">PRIVACY</a> |
					<a href="#">OUR TEAMS</a>
				</h3>
				<p>Copyright by phototime - all rights reserved</p>				
			</div>
			<div class="footer_logo">
			</div>
			

		</footer>
	</div>
</body>
</html>