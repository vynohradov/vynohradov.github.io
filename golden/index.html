<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1 ">
    <!--meta name="format-detection" content="telephone=no"-->
    <link rel="stylesheet" href="css/style.css" />
    <title>Golden</title>

</head>

<body>
    <div class="wrapper">
        <section class="title_section_1">
            <header>
                <div class="header_logo">golden</div>

                <nav class="menu_block">
                    <ul>
                        <li class="disable current"><a href="#">menu</a></li>
                        <li><a href="#">home</a></li>
                        <li><a href="#services">services</a></li>
                        <li><a href="#portfolio">portfolio</a></li>
                        <li><a href="#about">about</a></li>
                        <li><a href="#contact">contact</a></li>
                    </ul>
                </nav>
            </header>
            <div class="header_content">
                <p>welcome to our studio!</p>
                <h1>it's nice to meet you</h1>
                <button>tell me more</button>
            </div>

        </section>
        <section id="services" class="services_frame_2">

            <h2>services</h2>
            <p>Lorem ipsum dolor sit amet consectetur.</p>

            <div id="cats">
                <div class="cat">
                    <div class="icon basket"></div>
                    <h3>e-commerce</h3>

                    <p>Lorem ipsum dolor sit amet, <br> adipisicing elit. Assumenda comm<br> hic in libero modi omnis.</p>
                </div>
                <div class="cat">
                    <div class="icon laptop"></div>
                    <h3>responsive web</h3>

                    <p>Lorem ipsum dolor sit amet, <br> adipisicing elit. Assumenda comm<br> hic in libero modi omnis.</p>
                </div>
                <div class="cat">
                    <div class="icon lock"></div>
                    <h3>web security</h3>

                    <p>Lorem ipsum dolor sit amet, <br> adipisicing elit. Assumenda comm<br> hic in libero modi omnis.</p>
                </div>
            </div>
        </section>

        <a id="portfolio"></a>
        <section class="portfolio_frame_3">
            <h2>our portfolio</h2>
            <p>Lorem ipsum dolor sit amet consectetur.</p>

            <div id="items">
                <div class="item">
                    <a href="#">
                        <img src="./img/frame_3/pics1_1.jpg" alt="">
                        <h4>ebony & ivory</h4>
                        <p>branding</p>
                    </a>
                </div>
                <div class="item">
                    <a href="#">
                        <img src="./img/frame_3/pics1_2.jpg" alt="">
                        <h4>smart stationary</h4>
                        <p>print design</p>
                    </a>
                </div>
                <div class="item">
                    <a href="#">
                        <img src="./img/frame_3/pics1_3.jpg" alt="">
                        <h4>clever paster</h4>
                        <p>print design</p>
                    </a>
                </div>
                <div class="item">
                    <a href="#">
                        <img src="./img/frame_3/pics2_1.jpg" alt="">
                        <h4>vinyl record</h4>
                        <p>product mock-up</p>
                    </a>
                </div>
                <div class="item">
                    <a href="#">
                        <img src="./img/frame_3/pics2_2.jpg" alt="">
                        <h4>treehouse template</h4>
                        <p>web design</p>
                    </a>
                </div>
                <div class="item">
                    <a href="#">
                        <img src="./img/frame_3/pics2_3.jpg" alt="">
                        <h4>burned</h4>
                        <p>branding</p>
                    </a>
                </div>
            </div>

        </section>
         <a id="about"></a>
        <section class="aboutus_frame_4">
            <h2>about us</h2>
            <p>Lorem ipsum dolor sit amet consectetur.</p>

            <div class="time_stamp">
                <div class="image ts_img_1">
                    <div class="description left_side">
                        <h4>july 2010</h4>
                        <h4>our humble beginnings</h4>

                        <p>Lorem ipsum dolor sit amet,<br> consectetur adipisicing elit.<br> Amet cum deserunt dolorum ea<br>                            eaque excepturi incidunt ipsam,<br> quibusdam quod repudiandae.</p>
                    </div>
                </div>
            </div>

            <div class="timeline"></div>

            <div class="time_stamp">
                <div class="image ts_img_2">
                    <div class="description right_side">
                        <h4>january 2011</h4>
                        <h4>facing startup battles</h4>

                        <p>Lorem ipsum dolor sit amet,<br> consectetur adipisicing elit.<br> Amet cum deserunt dolorum ea<br>                            eaque excepturi incidunt ipsam,<br> quibusdam quod repudiandae.</p>
                    </div>
                </div>
            </div>

            <div class="timeline"></div>

            <div class="time_stamp">
                <div class="image ts_img_3">
                    <div class="description left_side">
                        <h4>december 2012</h4>
                        <h4>enter the dark days</h4>

                        <p>Lorem ipsum dolor sit amet,<br> consectetur adipisicing elit.<br> Amet cum deserunt dolorum ea<br>                            eaque excepturi incidunt ipsam,<br> quibusdam quod repudiandae.</p>
                    </div>
                </div>
            </div>

            <div class="timeline"></div>

            <div class="time_stamp">
                <div class="image ts_img_4">
                    <div class="description right_side">
                        <h4>february 2014</h4>
                        <h4>our triumph</h4>

                        <p>Lorem ipsum dolor sit amet,<br> consectetur adipisicing elit.<br> Amet cum deserunt dolorum ea<br>                            eaque excepturi incidunt ipsam,<br> quibusdam quod repudiandae.</p>
                    </div>
                </div>

            </div>

            <div class="timeline"></div>

            <div class="time_stamp">
                <div class="image">
                    <div class="description inside">
                        <h4>our <br> story<br> continues<br> ...</h4>

                    </div>
                </div>

            </div>

        </section>

        <section class="team_frame_5">
            <h2>our amazing team</h2>
            <p>Lorem ipsum dolor sit amet consectetur.</p>

            <div id="stuff">
                <div class="employee">
                    <div class="photo"></div>
                    <div class="name">
                        <h4>kimberly thompson</h4>
                        <h5>marketer</h5>
                    </div>
                    <div class="social_networks">
                        <div class="sn_icon twitter twitter_hover"></div>
                        <div class="sn_icon facebook facebook_hover"></div>
                        <div class="sn_icon pinterest pinterest_hover"></div>
                    </div>
                </div>
                <div class="employee">
                    <div class="photo"></div>
                    <div class="name">
                        <h4>rico massimo</h4>
                        <h5>coder</h5>
                    </div>
                    <div class="social_networks">
                        <div class="sn_icon twitter twitter_hover"></div>
                        <div class="sn_icon facebook facebook_hover"></div>
                        <div class="sn_icon google_plus google_plus_hover"></div>
                    </div>
                </div>
                <div class="employee">
                    <div class="photo"></div>
                    <div class="name">
                        <h4>uku mason</h4>
                        <h5>graphic designer</h5>
                    </div>
                    <div class="social_networks">
                        <div class="sn_icon twitter twitter_hover"></div>
                        <div class="sn_icon facebook facebook_hover"></div>
                        <div class="sn_icon pinterest pinterest_hover"></div>
                    </div>
                </div>
            </div>
            <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.<br> Ad adipisci animi culpa, cumque doloribus eius
                fuga fugiat.
            </p>
        </section>
        <section class="partners_frame_6">
            <img src="img/partners_logos/envato.png" alt="Envato logo" />
            <img src="img/partners_logos/wordpress.png" alt="WordPress logo" />
            <img src="img/partners_logos/tuts.png" alt="Tuts+ logo" />
            <img src="img/partners_logos/microlance.png" alt="Microlancer logo" />
        </section>
        <a id="contact"></a>
        <section class="contactus_frame_7">
            <h2>contact us</h2>
            <p>Lorem ipsum dolor sit amet consectetur.</p>

            <form action="#">
                <div class="col_1">
                    <input type="text" placeholder="your name *" />
                    <input type="email" placeholder="your e-mail *" />
                    <input type="text" placeholder="subject" />
                </div>
                <div class="col_2">
                    <textarea placeholder="your message *"></textarea>
                </div>
                <button>send message</button>
            </form>
        </section>

        <footer>
            <p>&copy Copyright 2014 FreebiesXpress.com</p>

            <div class="social_networks">
                <div class="sn_icon twitter_inverse twitter_hover"></div>
                <div class="sn_icon facebook_inverse facebook_hover"></div>
                <div class="sn_icon pinterest_inverse pinterest_hover"></div>
                <div class="sn_icon google_plus_inverse google_plus_hover"></div>

            </div>
        </footer>
    </div>
</body>

</html>